src/e2e-test/resources/pluginParameters.properties

```diff
@@ -1,6 +1,16 @@
 projectId=cdf-athena
+partitionFrom=2024-12-03
```

Comment:
Removed and add all the BigQuery related changes in BigQuery properties section only